.betterer.results

```diff
@@ -3051,6 +3051,10 @@ exports[`better eslint`] = {
     "public/app/features/canvas/elements/notFound.tsx:5381": [
       [0, 0, 0, "Unexpected any. Specify a different type.", "0"]
     ],
+    "public/app/features/canvas/elements/server.tsx:5381": [
+      [0, 0, 0, "Unexpected any. Specify a different type.", "0"],
+      [0, 0, 0, "Unexpected any. Specify a different type.", "1"]
+    ],
```

Comment:
Can we find a more appropriate type to pass to `CanvasElementItem` instead of `any`? Or use a eslint ignore comment. 

The intention of Betterer is to fix our existing violations of lint rules we agreed to add, and remove the tool. We can't do that if we keep adding more exceptions.